apps/pie-docs/test/visual/navigation.spec.js

```diff
@@ -1,5 +1,5 @@
+import { test } from '@playwright/test';
```

Comment:
is it expected to import the test from "@playwright/test" directly here? 
the previous file imported it from `@justeattakeaway/pie-webc-testing/src/playwright/playwright-fixtures` and one of them looks inconsistent

Reply:
Yep. the custom fixture was only needed for a11y tests (to expose the `makeAxeBuilder` property). Whoever wrote this spec originally probably copied the a11y spec and didn't update the import, that's all!